Added Registries to Remove for Manual Tentacle Uninstallation (#1348)
OctopusDeploy/docs: docs/administration/managing-infrastructure/tentacle-configuration-and-file-storage/manually-uninstall-tentacle.md

:::problem
This will also remove your deployed applications if you have not configured Tentacle to use a different Application folder. See below for more details.
:::
4. Find and delete the Octopus registry entries from the following locations:
    * **`HKLM\SOFTWARE\Octopus`** 

>     * **`HKLM\SOFTWARE\Classes\Installer\Products`**

Reviewer: It may be good to explain how to determine which of the entries under these sections are to be deleted, to make sure they don't delete the entire (say) Products keys - especially given that the first line specifies the key to delete, while the last three have data inside them that needs to be deleted.  Does that make sense?

Reviewer 2: I'd agree with @MissedTheMark here. Looking at a local Octopus Server install which also has a Tentacle installed on it, I'd be wary of deleting `HKLM\SOFTWARE\Octopus` since it also contains the Octopus Server product:

![image](https://user-images.githubusercontent.com/1418993/132651629-5458a541-c468-43a1-a4e9-43729cc19e87.png)

I also think this page could do with a warning somewhere about the damage removing entries from the registry can do and that someone should ensure they are comfortable with the risks, At the very least the keys should be exported as a backup :smile:

Author: Thank you both - Great points!

Changes have been made but happy to edit further if you'd like :D

:::problem
This will also remove your deployed applications if you have not configured Tentacle to use a different Application folder. See below for more details.
:::

4. Find and delete the Octopus Tentacle registry entries from the following locations:
Entire Folder:
    * **`HKLM\SOFTWARE\Octopus\Tentacle`**

> Octopus Tentacle Specific Registries - Check in each folder for a key with DisplayName "Octopus Tetacle" and remove the folder that key resides in.

Reviewer 2: ```suggestion
Octopus Tentacle Specific Registries - Check in each folder for a key with a **DisplayName** of `Octopus Tentacle` and remove the folder where that key resides.
```

    * **`HKLM\SOFTWARE\Classes\Installer\Products`** 

>     * **`HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Products\<RandomID>`** - Look under InstallProperties for DisplayName "Octopus Tetacle"

Reviewer 2: ```suggestion
    * **`HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Products\<RandomID>`** - Look under InstallProperties for a **DisplayName** of  `Octopus Tentacle`.
```
